### PR TITLE
Re-adds the Disk Compartmentalizer

### PR DESCRIPTION
**Pull request by LtDiggs**
## What Does This PR Do
Brings back the ability to make Disk Compartmentalizers before their removal. This PR does not re-add them to the maps.
## Why It's Good For The Game
It was removed in https://github.com/ParadiseSS13/Paradise/pull/25370 as it was thought to be obsolete due to the DNA manipulator being able to hold Botany disks. Apparently it wasn't known, but the Disk Compartmentalizer also holds other disks such as Technology, Genetic, Design and Training. Removing it from the code had removed a suitable, easy and clean way of storing any disks.
## Testing

Made a Smartfridge board via Circuit Imprinter, used a screwdriver to change it to a Disk Compartmentalizer, built the machine and stored disks of varying types inside.
<hr>

### Declaration
- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/feature_freeze/CODE_OF_CONDUCT.md#currently-changes-to-the-following-types-of-content-reuires-pre-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
<hr>

## Changelog
:cl:
add: Restores the possibility of building the Disk Compartmentalizer
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm [hunk 1]
```diff
@@ -694,6 +694,40 @@
...
+	icon_lightmask = "disktoaster"
+	pass_flags = PASSTABLE
+	visible_contents = TRUE
+	board_type = /obj/machinery/smartfridge/disks
+
+/obj/machinery/smartfridge/disks/Initialize(mapload)
+	. = ..()
+	accepted_items_typecache = typecacheof(list(
+		/obj/item/disk,
+	))
```
Comment (1080pCat) [inline code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm:716]:
```suggestion
		/obj/item/disk
		))
```

**Comment by LtDiggs** [comment]
Re-Opened (#26380) and tested again due to the end of the Feature Freeze.

File: code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm [hunk 2]
```diff
@@ -694,6 +694,40 @@
...
+	icon_lightmask = "disktoaster"
+	pass_flags = PASSTABLE
+	visible_contents = TRUE
+	board_type = /obj/machinery/smartfridge/disks
+
+/obj/machinery/smartfridge/disks/Initialize(mapload)
+	. = ..()
+	accepted_items_typecache = typecacheof(list(
+		/obj/item/disk
+		))
```
Comment (Sirryan2002) [inline code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm:716]:
```suggestion
	))
```

**Review by Sirryan2002 — APPROVED** [review]

**Review by Miraviel — APPROVED** [review]

**Comment by EmeraldCandy** [comment]
I feel this is now especially important if the RnD rework goes through and how physical backups will be even more important

**Review by DGamerL — APPROVED** [review]

**Comment by FunnyMan3595** [comment]
> Made a Smartfridge board via Circuit Imprinter, used a screwdriver to change it to a Disk Compartmentalizer, built the machine and stored disks of varying types inside.

You make it with a SMARTFRIDGE board? No wonder I never had a clue how to make one outside of mapping.

**Pull request merged**